test/adapters/IDB_test.js

```diff
@@ -0,0 +1,214 @@
+"use strict";
+
+import chai, { expect } from "chai";
+import chaiAsPromised from "chai-as-promised";
+import sinon from "sinon";
+
+import IDB from "../../src/adapters/IDB.js";
+
+chai.use(chaiAsPromised);
+chai.should();
+chai.config.includeStack = true;
+
+const root = typeof window === "object" ? window : global;
+
+describe("adapters.IDB", () => {
```

Comment:
I'll do that as soon as I'll add a localStorage adapter; that PR is quite large enough already :)